### DISCUSSION TITLE
Attach a callback after tests

### DISCUSSION THREAD

**Discussion by axieax** [mfussenegger/nvim-jdtls#261] [Q&A] [answered]
Hi, is there a way to attach a function (for retrieving `[dap-repl]` content) to run after `jdtls.test_class` or `jdtls.test_nearest_method` results are populated in the `[dap-repl]`? I'm currently deferring a call of this function after 2 seconds, where it has to search all buffers for one with the expected title, then extract its contents. Thanks!

**Comment by mfussenegger** [comment, marked as answer]
There is currently no such callback, but I'd be open to add one.

**Reply by axieax** [reply]
That will be extremely helpful! I'll open up an issue now :D
